gradle.properties

```diff
@@ -1,3 +1,6 @@
+android.defaults.buildfeatures.buildconfig=true
 android.enableJetifier=true
+android.nonFinalResIds=false
+android.nonTransitiveRClass=false
```

Comment:
same here: did that come by some automation?

Reply:
Res IDs are non-final now in recent gradles, you probably also saw some warning in android studio where we have switch statements with branches using menu IDs etc. saying that such Res IDs are no final etc. it seems this setting here disables the dynamic Res IDs avoiding issues for our usage,

this change was done automatically by AndroidStudio